internal/konnect/config_synchronizer.go

```diff
@@ -18,8 +18,12 @@ import (
 	"github.com/kong/kubernetes-ingress-controller/v3/internal/metrics"
 )
 
-// DefaultConfigUploadPeriod is the default period between operations to upload Kong configuration to Konnect.
-var DefaultConfigUploadPeriod = 30 * time.Second
+const (
+	// MinConfigUploadPeriod is the minimum period between operations to upload Kong configuration to Konnect.
+	MinConfigUploadPeriod = 10 * time.Second
+	// DefaultConfigUploadPeriod is the default period between operations to upload Kong configuration to Konnect.
+	DefaultConfigUploadPeriod = 30 * time.Second
```

Comment:
This will change the default period we upload config to Konnect (before the changes it was equal to the 3s sync loop period). Is it accepted by the product? If so, I think we should clearly state in the changelog this will change from 3s to 30s and can be reduced minimally to 10s by setting an env var.

Reply:
Added changelog entry.